rem.go

```diff
@@ -404,18 +423,33 @@ func promptBool(promptStr string) (yes bool) {
 	return true
 }
 
+// Try to find the position of the given argument, the posistioon of the first
+// instance is returned.
 func argsHaveOption(long string, short string) (hasOption bool, foundAt int) {
-	for i, arg := range os.Args {
-		if arg == "--"+long || arg == "-"+short {
-			return true, i
+	hasOptionShort, foundAtShort := argsHaveOptionExplicit("-" + short)
+	hasOptionLong, foundAtLong := argsHaveOptionExplicit("--" + long)
+	if hasOptionShort && hasOptionLong {
+		if foundAtLong < foundAtShort {
+			return true, foundAtLong
 		}
+		return true, foundAtShort
 	}
-	return false, 0
+	if hasOptionShort {
+		return true, foundAtShort
+	}
+	return hasOptionLong, foundAtLong
 }
 
 func argsHaveOptionLong(long string) (hasOption bool, foundAt int) {
+	return argsHaveOptionExplicit("--" + long)
+}
+
+func argsHaveOptionExplicit(argFull string) (hasOption bool, foundAt int) {
 	for i, arg := range os.Args {
-		if arg == "--"+long {
+		if arg == "--" {
+			return false, 0
+		}
+		if arg == argFull {
```

Comment:
IMO this is overly complicated. It's okay to have two separate functions for long args and then one for short and long args that both loop through `os.Args`